src/sparseml/optim/modifier.py

```diff
@@ -342,6 +348,99 @@ def yaml_key(clazz, framework: Union[str, None] = None):
 
         return "!{}.{}".format(framework, clazz.__name__)
 
+    @staticmethod
+    def comparator(one, two) -> int:
```

Comment:
should get a test for this in. am ok with later as tech debt since this req doesn't seem immediately breaking

Reply:
Agree on the test, but let's push that off for later -- there are some integration tests we'll need to do between pruning and quantization to make sure they order correctly based on this